packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -56,19 +61,60 @@ export const TokensList: React.FC<TokensListProps> = (props) => {
         }}
       />
       <Content>
-        <Box paddingBottom="space80">
-          <Label htmlFor="test" id="test-label">
-            Filter tokens
-          </Label>
-          <Input
-            type="text"
-            id="test"
-            aria-labelledby="test-label"
-            onChange={handleInput}
-            insertBefore={<FilterIcon decorative={false} title="Description of icon" />}
-          />
+        <Box marginBottom="space80">
+          <Grid gutter="space40">
+            <Column span={6}>
+              <Label htmlFor="test" id="test-label">
+                Filter tokens
+              </Label>
+              <Input
+                type="text"
+                id="test"
+                aria-labelledby="test-label"
+                onChange={handleInput}
+                insertBefore={<FilterIcon decorative={false} title="Description of icon" />}
+                placeholder="Filter by token name or value"
+              />
+            </Column>
+            <Column span={3}>
+              <Label htmlFor="theme-control" id="theme-control-label">
+                Theme
+              </Label>
+              <Select
+                id="theme-control"
+                defaultValue={theme === 'dark' ? 'dark' : 'default'}
+                onChange={(evt) => {
+                  if (evt.target.value === 'dark') {
+                    setSelectedTheme('dark');
+                    setTokens(DarkModeTokens.tokens);
+                  }
+                  if (evt.target.value === 'default') {
+                    setSelectedTheme('default');
+                    setTokens(Tokens.tokens);
+                  }
+                }}
+              >
+                <Option value="default">Default</Option>
+                <Option value="dark">Dark</Option>
+              </Select>
+            </Column>
+            <Column span={3}>
+              <Label htmlFor="format-control" id="format-control-label">
+                Format
+              </Label>
+              <Select
+                id="format-control"
+                onChange={(evt) => {
+                  if (evt.target.value === 'javascript') setUseJavascriptNames(true);
+                  else setUseJavascriptNames(false);
```

Comment:
question: should this selection be persistent? i.e. I select JavaScript format, and when I come back to the page in a different session it remembers that I selected that preferred format last time and automatically shows me what I want?

Reply:
I think we do want this and the theme to persist, I'm adding it to local storage. changes pushed up shortly